### PR TITLE
Jd5 za 47

### PR DESCRIPTION
**Pull request by czarnykapsel**

### CODE REVIEW

File: crypto-analyzer-web/src/main/java/com/infoshareacademy/zajavka/web/InfoServlet.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+@Inject
+    TemplateProvider templateProvider;
+
+    private static final Logger LOG = LoggerFactory.getLogger(DataUploadServlet.class);
+    private static final String TEMPLATE_NAME = "info";
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws SecurityException, IOException {
```
Comment (isacademyci) [inline crypto-analyzer-web/src/main/java/com/infoshareacademy/zajavka/web/InfoServlet.java]: MINOR Remove the declaration of thrown exception 'java.lang.SecurityException' which is a runtime exception. [rule](http://localhost:9000/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

**Comment by isacademyci** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: crypto-analyzer-web/src/main/java/com/infoshareacademy/zajavka/web/InfoServlet.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+@WebServlet(urlPatterns = "/info")
+public class InfoServlet extends HttpServlet {
+
+@Inject
```
Comment (MichalNowakowski) [inline crypto-analyzer-web/src/main/java/com/infoshareacademy/zajavka/web/InfoServlet.java]: formatowanie kodu - popraw pls

File: crypto-analyzer-web/src/main/java/com/infoshareacademy/zajavka/web/InfoServlet.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    TemplateProvider templateProvider;
+
+    private static final Logger LOG = LoggerFactory.getLogger(DataUploadServlet.class);
```
Comment (MichalNowakowski) [inline crypto-analyzer-web/src/main/java/com/infoshareacademy/zajavka/web/InfoServlet.java]: zla nazwa loggera

File: crypto-analyzer-web/src/main/java/com/infoshareacademy/zajavka/web/InfoServlet.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        Template template = templateProvider.getTemplate(getServletContext(), TEMPLATE_NAME);
+
+        try {
+            template.process(model, resp.getWriter());
+        } catch (TemplateException e) {
+            LOG.error("Error while processing the template: " + e);
+        }
+
+    }
+    }
```
Comment (MichalNowakowski) [inline crypto-analyzer-web/src/main/java/com/infoshareacademy/zajavka/web/InfoServlet.java]: formatowanie

**Pull request merged**